commands/command.go

```diff
@@ -262,3 +269,67 @@ func checkArgValue(v string, found bool, def Argument) error {
 func ClientError(msg string) error {
 	return &Error{Code: ErrClient, Message: msg}
 }
+
+func ExternalBinary() *Command {
+	return &Command{
+		Arguments: []Argument{
+			StringArg("args", false, true, "arguments for subcommand"),
+		},
+		External: true,
+		Run: func(req Request, res Response) {
+			binname := strings.Join(append([]string{"ipfs"}, req.Path()...), "-")
+			_, err := exec.LookPath(binname)
+			if err != nil {
+				// special case for '--help' on uninstalled binaries.
+				if req.Arguments()[0] == "--help" {
```

Comment:
it may not be the first argument. should check any of the args. maybe also for `-h`